model/model_training/trainer_rm.py

```diff
@@ -158,7 +160,7 @@ def argument_parsing(notebook=False, notebook_args=None):
     conf["wandb_entity"] = args.wandb_entity
     conf["local_rank"] = args.local_rank
     conf["deepspeed"] = args.deepspeed
-    if args.rng_seed is not None:
+    if "rng_seed" not in conf and args.rng_seed is not None:
```

Comment:
I don't think this logic is correct, but I can correct it after merge in the improve_small_rm branch..

Reply:
its my bad, I have reverted back